ui/src/index.css

```diff
@@ -10,4 +10,4 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
-}
+}
```

Comment:
What editor do you use that keeps removing the final newline?  Does it have a setting to change that?

Reply:
vs code, I just use whatever default web formatting it has